src/activities/tangram/Tangram.qml

```diff
@@ -132,8 +132,8 @@ ActivityBase {
                     sourceSize.width: modelData.width * background.playWidth
                     sourceSize.height: modelData.height * background.playWidth
                     z: index
-                    rotation: modelData.flipping ? 360 - modelData.rotation : modelData.rotation
-                    mirror: modelData.flipping ? true : false
+                    rotation: modelData.rotation
+                    mirror: modelData.flipping ? true : false
```

Comment:
can you replace the character with a normal space? I'm not sure on how it went there on the first place but let's use normal spaces.

Reply:
it comes whenever i change the line above it in qt creator. Used other editor and didn't happen. :/